packages/dev/core/src/Audio/audioSceneComponent.ts

```diff
@@ -217,7 +217,7 @@ Object.defineProperty(Scene.prototype, "audioListenerPositionProvider", {
             this._addComponent(compo);
         }
 
-        if (typeof value !== "function") {
+        if (value && typeof value !== "function") {

```

Comment:
I am not sure how this will change the current behavior?
typeof null === 'object', so it will anyhow fail. And any value with type function will be true-y.

Reply:
Doesn't this make sure `value` is *not* `null` before checking its type? If `value` *is* `null` then the type is not checked, which is what we want in this case, so we take the `else` branch and set `compo.audioListenerPositionProvider = value;`?

Reply:
Also note that this does fix the issue when tested against the playground referenced in the forum post: https://playground.babylonjs.com/#CBGEQX#1612